src/style.css

```diff
@@ -22,6 +21,38 @@ td {
   border-radius: 50%;
 }
 
+#topLeftCell {
```

Comment:
same - I think there's a better way to do this; somehow concatenate the two if it's top and left, etc.